src/App.vue

```diff
@@ -261,4 +305,18 @@ hr {
 .light .simplebar-scrollbar::before {
   background-color: rgba(0, 0, 0, 0.5);
 }
+
+/* Detect Theme */
+@media (prefers-color-scheme: light) {
+  body {
+    background: #fff;
+    color: #333;
+  }
+}
+@media (prefers-color-scheme: dark) {
+  body {
+    background: #333;
+    color: #fff;
+  }
+}
```

Comment:
We don't need these, because the classes added in the methods above will automatically apply the correct colors.
```suggestion
```